src/equations/endurance.js

```diff
@@ -32,9 +32,12 @@ export default class Config {
     label: "Years Before Broke",
     type: "text"
   }
-  static equation = ({principal, burnAmount, totalInterest, deposits} = {}) => {
+  static equation = ({principal, burnAmount, burnPeriod, totalInterest, deposits} = {}) => {
     totalInterest = 0.0;
     deposits = 0.0;
+    if (burnPeriod === 30) {
+      burnAmount = burnAmount * 12;
+    }
```

Comment:
hate this if statement.  This is trying to do something with months?

Reply:
just wanted to make sure making a selection with the new input was updating the result. we can fix this later